### DISCUSSION TITLE
Does srsRAN Project support NI USRP 2920?

### DISCUSSION THREAD

**Discussion by MalekAbouZeinab** [srsran/srsRAN_Project#231] [General Help] [answered]
I do have USRP 2920 and i need to try to implement the srsRAN project using it. Does it support the USRP 2920 ?

**Comment by brendan-mcauliffe** [comment]
Hi @MalekAbouZeinab, the USRP 2920 should work fine once you have the correct version of UHD. We haven't tested the 2920 ourselves, so you feel free to report issues if you come across any.

**Reply by MalekAbouZeinab** [reply]
thanks Brendan, is there a specific UHD i should have for it ?

**Reply by brendan-mcauliffe** [reply, marked as answer]
You can check [this section](https://docs.srsran.com/projects/project/en/latest/user_manuals/source/installation.html?highlight=uhd#rf-drivers) of the installation guide.
